gateway/modules/crud/operations.go

```diff
@@ -21,6 +24,18 @@ func (m *Module) Create(ctx context.Context, dbAlias, project, col string, req *
 		return err
 	}
 
+	// add the request for batch operation
+	if req.IsBatch {
+		ch, ok := m.batchMapTableToChan[dbAlias][col] // get channel for specified table
+		if !ok {
+			logrus.Errorf("error converting insert request to batch request unable to find channel for database %s & collection %s", dbAlias, col)
+			return fmt.Errorf("cannot find channel for database %s & collection %s", dbAlias, col)
+		}
+		// TODO VALIDATION OF PROJECT
+		ch.request <- batchRequest{document: req.Document, project: project}
+		return <-ch.response
```

Comment:
You'll get the response in the new channel you just created. Don't forget to close the channel in differ